src/Faithlife.Tracing.Zipkin/ZipkinTraceFactory.cs

```diff
@@ -10,16 +9,27 @@ namespace Faithlife.Tracing.Zipkin
 {
 	public sealed class ZipkinTraceFactory : ITraceFactory
 	{
-		public ZipkinTraceFactory(string collectorUrl)
+		public static ITraceFactory Create(Uri collectorUri) => collectorUri == null ? null : new ZipkinTraceFactory(collectorUri);
```

Comment:
Why shouldn't this fail on a null URI?

Reply:
It's a convenience for ASP.NET clients that may have the URL set to `null` for certain environments.

Reply:
Smells bad to me. That's not hard code for them to write.